lib/diagnostic.js

```diff
@@ -15,9 +15,18 @@
 
 const fs = require('fs'); // jshint ignore: line
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+let fileName = process.argv[2];
+
+let sum = 0;
+let i = 0;
+
+const sumLines = (fileName, callback) => {
+  fs.readFileSync(process.argv[2]).toString().split("\n");
+
+
+
+
+  callback(new Error('line not a number'));
```

Comment:
This line is fine, except you would need to wrap this in some kind of if-statement to determine that there is indeed an error.